src/services/dealer-price/index.ts

```diff
@@ -95,7 +95,7 @@ export const DealerPriceService = (): IDealerPriceService => {
           amountInSatoshis,
         ),
       )
-      return toCents(response.getAmountInCents())
+      return toCents(Math.floor(response.getAmountInCents()))
```

Comment:
if we will persist big int then should be parsed here

Reply:
Are you suggesting returning a UsdPaymentAmount? That change could be made but would expand the scope of the PR.